src/containers/Header/PreviewFlag.js

```diff
@@ -1,8 +1,20 @@
 import React from 'react';
+import {useTranslation} from "react-i18next";
```

Comment:
@queuing4oranges Add spaces to the curly brackets  - `import { useTranslation } from "react-i18next";`